core/controllers/classroom.py

```diff
@@ -62,7 +53,7 @@ def get(self, classroom_name):
 
         classroom_name_is_valid = False
         for classroom_dict in config_domain.TOPIC_IDS_FOR_CLASSROOM_PAGES.value:
-            if classroom_dict['name'] == classroom_name:
+            if classroom_dict['name'].lower() == classroom_name:
```

Comment:
Are we planning to do a hard redirect so that the url in the url bar also changes correctly -- e.g. /Math to /math? We might want to do that so that the /math URL becomes canonical.

Reply:
I've made changes so that even if `/Math` or `/mAtH` is entered, the user would be redirected to `/math` and the URL in the URL bar would also change. Basically, the classroom URL won't be case sensitive.